### PR TITLE
Preparation

### PR DESCRIPTION
**Pull request by macfrei**
Please have a look 🙂

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { loadFromLocal, saveToLocal } from '../lib/localStorage'
```
Comment (miriamhaenle) [inline src/components/App.js]: I think the App should not be part of the components folder but rather directly in the src folder (as the pivot point for everything)

File: src/components/App.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import { useEffect, useState } from 'react'
+import { v4 as uuidv4 } from 'uuid'
+import AppForm from './AppForm'
+import HomePage from './HomePage'
+import styled from 'styled-components/macro'
```
Comment (miriamhaenle) [inline src/components/App.js]:
```suggestion
import { useEffect, useState } from 'react'
import { v4 as uuidv4 } from 'uuid'
import styled from 'styled-components/macro'
import AppForm from './AppForm'
import HomePage from './HomePage'
```

File: src/components/HomePage.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+export default function HomePage({ todos }) {
```
Comment (miriamhaenle) [inline src/components/HomePage.js]: In a `pages` folder?

**Review by miriamhaenle — COMMENTED** [review]
Just a few comments but lgtm :)

File: src/components/App.js (hunk 1 above)
Comment (macfrei) [reply src/components/App.js]: Good point.

**Pull request merged**